docling/pipeline/vlm_pipeline.py

```diff
@@ -194,6 +246,48 @@ def _assemble_document(self, conv_res: ConversionResult) -> ConversionResult:
 
         return conv_res
 
+    def _turn_dt_into_doc(self, conv_res) -> DoclingDocument:
+        doctags_list = []
+        image_list = []
+        for page in conv_res.pages:
+            predicted_doctags = ""
+            img = PILImage.new("RGB", (1, 1), "rgb(255,255,255)")
+            if page.predictions.vlm_response:
+                predicted_doctags = page.predictions.vlm_response.text
+            if page.image:
+                img = page.image
+            image_list.append(img)
+            doctags_list.append(predicted_doctags)
+
+        doctags_list_c = cast(List[Union[Path, str]], doctags_list)
+        image_list_c = cast(List[Union[Path, PILImage.Image]], image_list)
+        doctags_doc = DocTagsDocument.from_doctags_and_image_pairs(
+            doctags_list_c, image_list_c
+        )
+        conv_res.document.load_from_doctags(doctags_doc)
```

Comment:
This should be changed to use static method `doc = DoclingDocument.load_from_doctags(...)`. This method is otherwise broken.

Reply:
good catch!